### PR TITLE
Slack appender config fix

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
## Context

Updating SlackAppender to read configuration params from config file. Was trying to read from env variables.
https://broadinstitute.atlassian.net/browse/DDP-5350

## Checklist

- [X ] I have labeled the type of changes involved using the `C-*` labels.
- [ X] I have assessed potential risks and labeled using the `R-*` labels.
- [X ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ X] I have considered security and privacy, and added `I-*` labels as needed
- [X ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [X ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [X ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

## FUD Score

_Overall, how are you feeling about these changes?_

- [ X] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?
Need to regenerate application.conf with api-build.sh

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ X] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/config/application.conf.ctmpl [hunk 1]
```diff
@@ -34,7 +34,9 @@
     },
     "slack": {
         "hook": "{{$slack.hook}}",
-        "channel": "{{$slack.channel}}"
+        "channel": "{{$slack.channel}}",
+        "queueSize": 10,
+        "intervalInMillis": 60
```
Comment (zyme) [inline pepper-apis/config/application.conf.ctmpl]: Should be 60,000?

Comment (MocanaAtBroad) [reply pepper-apis/config/application.conf.ctmpl]: Yes! 7b37acfb

**Review by zyme — APPROVED** [review]
Woo hoo!

File: pepper-apis/src/main/java/org/broadinstitute/ddp/log/SlackAppender.java [hunk 2]
```diff
@@ -109,17 +109,35 @@ private void init(String slackHookUrl,
      * Constructor used by deployments via logback.xml
      */
     public SlackAppender() {
-        init(System.getenv("SLACK_HOOK"),
-                System.getenv("SLACK_CHANNEL"),
-                System.getenv("SLACK_QUEUE_SIZE"),
-                System.getenv("SLACK_INTERVAL"));
```
Comment (yufengwng) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/log/SlackAppender.java:115]: Ah ha! No wonder I couldn't find where we're configuring the `SlackAppender`. I carelessly thought that we intentionally removed it!

@pegahtah: I lied. We're still using `SlackAppender` and it's configured from config file.

**Pull request merged**